R/import_mzroll.R

```diff
@@ -168,6 +168,163 @@ process_mzroll <- function(mzroll_db_path,
   return(mzroll_list)
 }
 
+#' Process Metabolon
+#' 
+#' Import results generated by metabolon into romic structure, using series of CSV
+#' output files delivered by Metabolon as inputs.
+#' 
+#' @param peak_areas path to peak areas CSV file from metabolon results,
+#' or table containing loaded peak areas
+#' @param chem_anno path to chemical annotations CSV file from metabolon results,
+#' or table containing loaded chemical annotations
+#' @param metadata_info path to metadata CSV file from metabolon results,
+#' or table containing loaded metadata
+#' 
+#' @param peak_floor_intensity value to use to replace NA peak measurements (missing values)
+#' 
+#' @examples
+#' install MetabolonR via remotes::install_github("miepstei/MetabolonR")
```

Comment:
this is failing R CMD check. Also, I don't think we should take on a dependency for an unmaintained github package. Can you either move this to tests as-is, or pull in just the files without installing the package?

Reply:
@shackett Sorry about the failing test - I didn't realize that `@examples` actually tries to execute the R code as a part of `R CMD check`.  Hopefully the corrected version will pass the test.